src/jit/lclvars.cpp

```diff
@@ -3444,6 +3445,17 @@ var_types LclVarDsc::lvaArgType()
                 break;
         }
     }
+#else  // FEATURE_UNIX_AMD64_STRUCT_PASSING
+    if (type == TYP_STRUCT)
+    {
+        NYI("lvaArgType");
+    }
+#endif // !FEATURE_UNIX_AMD64_STRUCT_PASSING
+#elif defined(_TARGET_ARM64_)
+    if (type == TYP_STRUCT)
+    {
+        NYI("lvaArgType");
+    }
 #elif defined(_TARGET_X86_)
```

Comment:
This string of `#ifdef`s begins with a negation, which can cause confusion. I think it would be better to put the other two cases first.